### PR TITLE
remove using namespace llvm

### PR DESCRIPTION
**Pull request by tgymnich**

### CODE REVIEW

**Review by wsmoses — APPROVED** [review]
Thank you this makes MLIR stuff so much easier.

File: enzyme/Enzyme/InstructionBatcher.cpp [hunk 1]
```diff
@@ -0,0 +1,279 @@
+//===- InstructionBatcher.h
```
Comment (wsmoses) [inline enzyme/Enzyme/InstructionBatcher.cpp:1]: .cpp ?

File: enzyme/Enzyme/TraceGenerator.cpp [hunk 2]
```diff
@@ -0,0 +1,195 @@
+#include "TraceGenerator.h"
```
Comment (wsmoses) [inline enzyme/Enzyme/TraceGenerator.cpp:1]: License Header?

File: enzyme/Enzyme/TraceInterface.cpp [hunk 3]
```diff
@@ -0,0 +1,242 @@
+#include "TraceInterface.h"
```
Comment (wsmoses) [inline enzyme/Enzyme/TraceInterface.cpp:1]: License header?

Comment (tgymnich) [reply enzyme/Enzyme/TraceInterface.cpp:1]: #1009

**Pull request merged**